local/blockchain.go

```diff
@@ -668,6 +663,7 @@ func (ln *localNetwork) waitSubnetValidators(
 	ln.log.Info(logging.Green.Wrap("waiting for the nodes to become subnet validators"))
 	for {
 		ready := true
+	subnetLoop:
```

Comment:
would prefer not to use labels in general

Reply:
Yeah, me too, but it's the only way to avoid doing unnecessary loop iterations

Reply:
You can use an `if !ready` and a simple `break` as the last sentence in the outer for

Reply:
done